scripts/mkcloud

```diff
@@ -973,6 +973,7 @@ function crowbarupgrade_5plus
         onadmin check_admin_server_upgraded
         # use crowbar-init to bootstrap crowbar (only for 6-7 upgrade)
         iscloudver 7 && onadmin bootstrapcrowbar "with_upgrade"
+        wait_for 30 10 "onadmin is_crowbar_api_available" "crowbar service to start"
```

Comment:
Have you considered just calling "onadmin wait_for_crowbar_api" ?

Reply:
I did not. I do not think it's that much different

Reply:
Oh, actually, you're right - it is better!